app/src/renderer/components/common/PageAbout.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <tm-page data-title="About" hide-header>
+    <div class="container">
+      <div class="card">
+        <h1>About</h1>
+        <p>
+          Lunie was the <b>first</b> visual interface ever build for the Cosmos
+          network. We develope Lunie putting the <b>user first</b>. We try to <b>
+            reduce the complexity
+          </b> of blockchain to minimum so you can <b>focus</b> on what you really
+          aim to do. We want to keep Lunie <b>Open Source</b> so the community can
+          grow from the learnings we have had already.
+          The developers of Lunie accompanied the Cosmos SDK team to build
+          the amazing
+          tooling that exist today for building visual interfaces for Cosmos.
+          We want to continue assisting developement in the network so we all
+          can benefit. We want to keep Lunie <b>independent</b> from validators so
+          there is an entity you can trust.
+        </p>
+      </div>
+      <div class="card">
+        <h1>Contact</h1>
```

Comment:
I'd just leave the email and maybe a public Telegram/Reddit/Discord/Riot channel to reach out. Also I'd be nice to have the pictures of the team with media profiles.

Reply:
Good ideas!